packages/KSelect/KSelect.vue

```diff
@@ -442,8 +443,8 @@ export default {
     }
 
     &.k-select-pop-dropdown {
-      --KPopPaddingY: var(--spacing-md);
-      --KPopPaddingX: var(--spacing-xxs);
+      --KPopPaddingY: 7px;
+      --KPopPaddingX: 7px;
       border: 1px solid var(--blue-200);
```

Comment:
@aanchalm01 I forgot to add this to the ticket; we want to change the border color of the popover to `grey-300`
```suggestion
      border: 1px solid var(--grey-300);
```